src/app/services/graph-constants.ts

```diff
@@ -1,8 +1,15 @@
 export const GRAPH_URL = 'https://graph.microsoft.com';
 export const USER_INFO_URL = `${GRAPH_URL}/v1.0/me`;
+export const USER_BETA_INFO_URL = `${GRAPH_URL}/beta/me/profile`;
```

Comment:
I don't see this const being used anywhere else in your PR, was this added intentionally?

Reply:
i was trying out random stuff, thanks for catching that LOL. will remove
![YikesMonkeyLookAndLeaveGIF](https://user-images.githubusercontent.com/55033656/123440031-3451a800-d587-11eb-9059-145f4c38085f.gif)